src/Map/map.js

```diff
@@ -91,9 +91,9 @@ export const getMap = (
           <Marker
             coordinate={{
               latitude:
-                addresses.length > 0 ? addresses[index].location.lat : 0,
+                addresses.length > 0 ? addresses[index] && addresses[index].location.lat : 0,
```

Comment:
```suggestion
                addresses.length > 0 && addresses[index] ? addresses[index].location.lat : 0,
```
Let's try to keep all of the testing logic in the first part of the ternary, keeps things cleaner.
And make sure to make the same change for the other lines too.